pandas/sparse/tests/test_sparse.py

```diff
@@ -30,6 +32,7 @@
 from pandas.sparse.api import (SparseSeries, SparseTimeSeries,
                                SparseDataFrame, SparsePanel,
                                SparseArray)
+import scipy.sparse
```

Comment:
Also here the same comment. There should be some skipping of the tests when there is no scipy. Search for `_skip_if_no_scipy()` to see how it is done elsewhere